### PR TITLE
Add graphs from perf-long/checkpoint-stress-test to jenkins/plots

### PR DESCRIPTION
**Pull request by sulabhM**

### CODE REVIEW

**Comment by sulabhM** [comment]
@agorrod : Please review

File: jenkins/plots/index.html [hunk 1]
```diff
@@ -28,15 +28,15 @@ <h2 id="stress-perf-build-status">Stress Perf <a href="http://build.wiredtiger.c
...
-<h2 id="ycsb-mongo-master-wt-mongodb-3.0-build-status">YCSB Mongo Master + WT mongodb-3.0 <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-mongodb-3.0/"><img src="http://build.wiredtiger.com:8080/buildStatus/icon?job=mongodb-perf-ycsb-mongodb-3.0" alt="Build Status" /></a></h2>
+<h2 id="ycsb-mongo-master-wt-mongodb-3.0-build-status">YCSB Mongo Master + WT mongodb-3.0 <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-mongodb-3.0/"><embed src="http://build.wiredtiger.com:8080/buildStatus/icon?job=mongodb-perf-ycsb-mongodb-3.0" /></a></h2>
```
Comment (agorrod) [inline jenkins/plots/index.html]: Do you understand why this change is here?

Comment (sulabhM) [reply jenkins/plots/index.html]: I didnt notice that one. It gets auto generated. 
Maybe different version of pandoc that gets used for auto generation.

**Comment by sulabhM** [comment]
@agorrod : Review this now, I removed those extra auto modified lines manually
